themes/solr/templates/operator/downloads.html

```diff
@@ -0,0 +1,160 @@
+{% extends "operator/page.html" %}
+
+{% block content_inner %}
+<div class="small-12 columns">
+  <style type="text/css">
+    .headerlink, .elementid-permalink {
+      visibility: hidden;
+    }
+    h2:hover > .headerlink, h3:hover > .headerlink, h1:hover > .headerlink, h6:hover > .headerlink, h4:hover > .headerlink, h5:hover > .headerlink, dt:hover > .elementid-permalink {
+      visibility: visible
+    }
+  </style>
+
+  <h1 id="solr-downloads">{{ page.title }}
+    <a class="headerlink" href="#solr-downloads" title="Permanent link">¶</a>
+  </h1>
+  {{ page.content }}
+  <h2 id="solr-{{ SOLR_OPERATOR_LATEST_RELEASE | replace(".", "") }}">Solr Operator {{ SOLR_OPERATOR_LATEST_RELEASE }}
+    <a class="headerlink" href="#solr-{{ SOLR_OPERATOR_LATEST_RELEASE | replace(".", "") }}" title="Permanent link">¶</a>
+  </h2>
+
+  <p>Solr Operator {{ SOLR_OPERATOR_LATEST_RELEASE }} is the most recent Apache Solr Operator release.</p>
+
+  <ul>
+    <li>Source release:
+      <a href="https://www.apache.org/dyn/closer.lua/solr/operator/{{ SOLR_OPERATOR_LATEST_RELEASE }}/solr-operator-{{ SOLR_OPERATOR_LATEST_RELEASE }}.tgz">solr-operator-{{ SOLR_OPERATOR_LATEST_RELEASE }}.tgz</a>
+      [<a href="https://downloads.apache.org/solr/operator/{{ SOLR_OPERATOR_LATEST_RELEASE }}/solr-operator-{{ SOLR_OPERATOR_LATEST_RELEASE }}.tgz.asc">PGP</a>]
+      [<a href="https://downloads.apache.org/solr/operator/{{ SOLR_OPERATOR_LATEST_RELEASE }}/solr-operator-{{ SOLR_OPERATOR_LATEST_RELEASE }}.tgz.sha512">SHA512</a>]
+    </li>
+
+    <li>Docker releases:
+      <a href="https://hub.docker.com/r/apache/solr-operator">apache/solr-operator</a>
+    </li>
+
+    <li>Helm Chart release:
+      <a href="https://artifacthub.io/packages/helm/apache-solr/solr-operator/{{ SOLR_OPERATOR_LATEST_RELEASE  | replace("v", "")  }}">Artifact Hub</a>
+    </li>
+
+    <li>CRDs:
+      <ul>
+        <li><a href="{{ SITEURL }}/operator/downloads/crds/{{ SOLR_OPERATOR_LATEST_RELEASE }}/all.yaml">All</a></li>
+        <li><a href="{{ SITEURL }}/operator/downloads/crds/{{ SOLR_OPERATOR_LATEST_RELEASE }}/solrclouds.yaml">Solr Cloud</a></li>
+        <li><a href="{{ SITEURL }}/operator/downloads/crds/{{ SOLR_OPERATOR_LATEST_RELEASE }}/solrprometheusexporters.yaml">Solr Prometheus Exporter</a></li>
+        <li><a href="{{ SITEURL }}/operator/downloads/crds/{{ SOLR_OPERATOR_LATEST_RELEASE }}/solrbackups.yaml">Solr Backup</a></li>
+      </ul>
+    </li>
+
+    <!-- TODO: Change log -->
+    <li><a href="{{ SITEURL }}/{{ SOLR_LATEST_RELEASE | replace(".", "_") }}/changes/Changes.html">Change log</a></li>
```

Comment:
Simly link to https://github.com/apache/solr-operator/releases ?

Reply:
Maybe so. I'm unsure of how to deal with the change list yet. We also need to generate a single bulleted changeList for the artifactHub page.

Reply:
I see. So as a start link to that releases page, and then change it once we have a better location? Or could artifactHub serve as the main changelog? Guess we need changes info in the git repo or other ASF controlled location as well.

Reply:
Yeah, I'll go with the releases page until we have another solution.